VERSIONING.md

```diff
@@ -0,0 +1,90 @@
+# Versioning
+
+This document describes the versioning policy for this repository. This policy
+is designed so that the following goal can be achieved:
+
+**Users are provided a codebase of value that is stable and secure.**
+
+## Policy
+
+* Versioning of this project will be idiomatic of a Go project using [Go
+  modules](https://github.com/golang/go/wiki/Modules).
+    * [Semantic import
+      versioning](https://github.com/golang/go/wiki/Modules#semantic-import-versioning)
+      will be used.
+        * Versions will comply with [semver 2.0](https://semver.org/spec/v2.0.0.html).
+        * If a module is version `v2` or higher, the major version of the module
+          must be included as a `/vN` at the end of the module paths used in
+          `go.mod` files (e.g., `module go.opentelemetry.io/collector/v2`, `require
+          go.opentelemetry.io/collector/v2 v2.0.1`) and in the package import path
+          (e.g., `import "go.opentelemetry.io/collector/v2/component"`). This includes the
+          paths used in `go get` commands (e.g., `go get
+          go.opentelemetry.io/collector/v2@v2.0.1`.  Note there is both a `/v2` and a
+          `@v2.0.1` in that example. One way to think about it is that the module
+          name now includes the `/v2`, so include `/v2` whenever you are using the
+          module name).
+        * If a module is version `v0` or `v1`, do not include the major version in
```

Comment:
I assume we will use 0.x.x for unstable versions. Maybe, it's good to note it here.